changedetectionio/tests/test_xpath_selector.py

```diff
@@ -277,3 +277,168 @@ def test_check_with_prefix_include_filters(client, live_server):
     assert b"Some text that will change" not in res.data #not in selector
 
     client.get(url_for("form_delete", uuid="all"), follow_redirects=True)
+
+def test_xpath_20(client, live_server):
+    res = client.get(url_for("form_delete", uuid="all"), follow_redirects=True)
+    assert b'Deleted' in res.data
+
+    # Give the endpoint time to spin up
+    time.sleep(1)
+
+    set_original_response()
+
+    # Add our URL to the import page
+    test_url = url_for('test_endpoint', _external=True)
+    res = client.post(
+        url_for("import_page"),
+        data={"urls": test_url},
+        follow_redirects=True
+    )
+    assert b"1 Imported" in res.data
+    time.sleep(3)
+
+    res = client.post(
+        url_for("edit_page", uuid="first"),
+        data={"include_filters": "//*[contains(@class, 'sametext')]|//*[contains(@class, 'changetext')]",
+              "url": test_url,
+              "tags": "",
+              "headers": "",
+              'fetch_backend': "html_requests"},
+        follow_redirects=True
+    )
+
+    assert b"Updated watch." in res.data
+    time.sleep(3)
+
+    res = client.get(
+        url_for("preview_page", uuid="first"),
+        follow_redirects=True
+    )
+
+    assert b"Some text thats the same" in res.data #in selector
+    assert b"Some text that will change" in res.data #in selector
+
+    client.get(url_for("form_delete", uuid="all"), follow_redirects=True)
+
+def test_xpath_20_function_count(client, live_server):
+    res = client.get(url_for("form_delete", uuid="all"), follow_redirects=True)
+    assert b'Deleted' in res.data
+
+    # Give the endpoint time to spin up
+    time.sleep(1)
+
+    set_original_response()
+
+    # Add our URL to the import page
+    test_url = url_for('test_endpoint', _external=True)
+    res = client.post(
+        url_for("import_page"),
+        data={"urls": test_url},
+        follow_redirects=True
+    )
+    assert b"1 Imported" in res.data
+    time.sleep(3)
+
+    res = client.post(
+        url_for("edit_page", uuid="first"),
+        data={"include_filters": "xpath:count(//div)",
+              "url": test_url,
+              "tags": "",
+              "headers": "",
+              'fetch_backend': "html_requests"},
+        follow_redirects=True
+    )
+
+    assert b"Updated watch." in res.data
+    time.sleep(3)
+
+    res = client.get(
+        url_for("preview_page", uuid="first"),
+        follow_redirects=True
+    )
+
+    assert b"2" in res.data #in selector
```

Comment:
hmmm this "2" could come from anywhere tho, in the html.. some embeded SVG or..

Reply:
No, it doesn't. `xpath:count(//div)` is count all `div` in the document. Look into the line 330 above. The code(https://github.com/Constantin1489/changedetection.io/blob/79cf9848e677fea0bfe3d3438e665e3e7103afd3/changedetectionio/tests/test_xpath_selector.py#L12) shows two div elements.

Reply:
@dgtlmoon Okay. Due to https://github.com/Constantin1489/changedetection.io/actions/runs/6133785669/job/16645925296#step:8:3469

Reply:
@dgtlmoon What about `b'<div class="">2</div>' in in res.data`?

Reply:
No, forget it, please. I will do `/html/body/count(div) * 123456789987654321`.

Reply:
> No, forget about my suggestion, please. I will do `/html/body/count(div) * 123456789987654321`.

Perfect, i'll wait for that